projects/testing-library/src/lib/testing-library.ts

```diff
@@ -386,7 +386,9 @@ function cleanupAtFixture(fixture: ComponentFixture<any>) {
 // this ensures that tests run in isolation from each other
 // if you don't like this, set the ATL_SKIP_AUTO_CLEANUP env variable to 'true'
 if (typeof process === 'undefined' || !process.env?.ATL_SKIP_AUTO_CLEANUP) {
+  // @ts-ignore - cannot find name 'afterEach'
   if (typeof afterEach === 'function') {
+    // @ts-ignore - cannot find name 'afterEach'
```

Comment:
Getting the Typescript Error, `cannot find name 'afterEach'`
Any suggestions how to fix this?

Reply:
Is this still required? I tried it locally, and I'm not required to add the comment.

Reply:
hmm.. strange.. Now it works without the `ts-ignore`
Removed 👍